src/pages/git/git-pull/index.md

```diff
@@ -3,13 +3,31 @@ title: Git Pull
 ---
 ## Git Pull
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/git/git-pull/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+[**git pull**](https://git-scm.com/docs/git-pull) allows you to get new commits from a remote repository. To do this, *git pull* fetches ( *git fetch* ) the new commits and merges [(*git merge*)](https://guide.freecodecamp.org/git/git-merge) these into your local branch.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+This command's syntax is as follows:
+```bash
+git pull [options] [<repository>]
+```
+in which:
+- **repository** is the URL to your repo. Example:  https://github.com/freeCodeCamp/freeCodeCamp.git
+- **options** are the command options, such as *--quiet* or *--verbose*. Learn more about the *options* in the [documentation](https://git-scm.com/docs/git-pull).
+- **refspec** specifies which refs to fetch and which local refs to update.
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+If you already have cloned a repository and want to **pull the latest commits**, all you'll have to do is:
 
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+```bash
+git pull
+```
+If you want to **pull a specific branch**, you can use:
+```bash
+git fetch <remote_name> <branch_name>
```

Comment:
Similar comment as above: `<remote_name>` -> `REMOTE-NAME` and `<branch_name>` to `BRANCH-NAME`, and update the bullets below to match.